dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/DependentTaskTest.java

```diff
@@ -37,188 +43,379 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationContext;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
It is not recommended to use MockitoJUnitRunner.Silent.class, can you modify it to MockitoJUnitRunner.class?

Reply:
Done. Is there any side effect?

Reply:
@CalvinKirs same usage on #3385, should I follow this one?

Reply:
No, your last submission has resolved it.

Reply:
> @CalvinKirs same usage on #3385, should I follow this one?

Very good, I recommend this. If you are interested, you can perfect it

Reply:
Don't make changes for now, I will run the test again tomorrow.

Reply:
Hello, temporarily use MockitoJUnitRunner.Silent.class to solve it. This problem may be caused by the same moc data defined elsewhere.

Reply:
Reverted. Hope this can be fixed soon.